File: 151004/Gonzarevich/lab1/Entity/Comment.php
<?php

namespace App\Entity;

class Comment
{

    private ?int $id = null;

    private ?int $issueId = null;

    private ?string $content = null;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function setId(int $id): static
    {
        $this->id = $id;
        return $this;
    }

    public function getIssueId(): ?int
    {
        return $this->issueId;
    }

    public function setIssueId(int $issueId): static
    {
        $this->issueId = $issueId;

        return $this;
    }

    public function getContent(): ?string
    {
        return $this->content;
    }

    public function setContent(string $content): static
    {
        $this->content = $content;

        return $this;
    }

    public function populate(array $args) {

        list($temp, $this->issueId, $this->content) = $args;
    }
}

File: 151001/Timoshek/discussion/src/main/java/by/bsuir/egor/Entity/Comment.java
package by.bsuir.egor.Entity;

import jakarta.persistence.*;
import org.springframework.data.cassandra.core.cql.PrimaryKeyType;
import org.springframework.data.cassandra.core.mapping.PrimaryKeyColumn;

import java.util.UUID;

@Entity
@Table(name="comment")
public class Comment {

    @PrimaryKeyColumn(name="country", ordinal = 0, type = PrimaryKeyType.PARTITIONED)
    private String country = "BY";

    @PrimaryKeyColumn(name="id",ordinal = 0,type = PrimaryKeyType.CLUSTERED)
    @Id
    private Long id = UUID.randomUUID().getLeastSignificantBits() % 1000000;

    @PrimaryKeyColumn(name="issue_id",ordinal = 0,type = PrimaryKeyType.CLUSTERED)
    private Long issueId;

    @Column(name = "content")
    private String content;

    public Comment() {

    }

    public Long getId() {
        return id;
    }

    public String getContent() {
        return content;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public String getCountry()
    {
        return this.country;
    }

    public void setCountry(String country)
    {
        this.country = country;
    }

    public Long getIssueId() {
        return issueId;
    }

    public void setIssueId(Long issueId) {
        this.issueId = issueId;
    }
}